fontforge/python.c

```diff
@@ -13294,6 +13296,7 @@ return( -1 );
 return( -1 );
     }
     if ( PyErr_Occurred() != NULL ) {
+        free(ss);
```

Comment:
This may not be clear from the code, but a SplineSet should never be "deallocated" with just `free()`. Use `SplinePointListsFree()`. It's NULL-tolerant, so it can be called as `free()` is, and I guess I'm fine calling it here.